src/Sulu/Component/HttpKernel/SuluKernel.php

```diff
@@ -82,6 +82,14 @@ protected function configureContainer(ContainerBuilder $container, LoaderInterfa
         $container->addResource(new FileResource($this->getProjectDir() . '/config/bundles.php'));
         $confDir = $this->getProjectDir() . '/config';
 
+        // this is a bc layer that sulu_admin.yaml is only loaded when bundle is registered
+        // this can be removed when there is no longer context based configuration
+        $bundles = $container->getParameter('kernel.bundles');
+        if (isset($bundles['SuluAdminBundle'])) {
```

Comment:
@nnatter this fixes your comment here: https://github.com/sulu/skeleton/pull/110#issuecomment-822359021